templates/system/html.html.twig

```diff
@@ -31,48 +31,34 @@
     <head-placeholder token="{{ placeholder_token }}">
     <title>{{ head_title|safe_join(' | ') }}</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="google-site-verification" content="whpL-pBc0Hexn-Gz9egGV8YJcNMHZxlFSf9CeNr99dc"/>
     <link rel="apple-touch-icon" sizes="180x180" href="/{{ base_path ~ directory }}/dist/assets/img/favicon/apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="/{{ base_path ~ directory }}/dist/assets/img/favicon/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/{{ base_path ~ directory }}/dist/assets/img/favicon/favicon-16x16.png">
     <link rel="manifest" href="/{{ base_path ~ directory }}/dist/assets/img/favicon/favicon.json">
     <link rel="mask-icon" href="/{{ base_path ~ directory }}/dist/assets/img/favicon/safari-pinned-tab.svg" color="#5bbad5">
     <meta name="theme-color" content="#000000">
-    {#<!-- Schema.org for Google -->#}
-    {#<meta itemprop="name" content="Croydon Council">#}
-    {#<meta itemprop="description" content="Delivering 24 hour council services to residents, visitors and businesses in Croydon">#}
-    {#<meta itemprop="image" content="">#}
-    <meta name="fb:admins" content="">
 
-    <link type="text/css" rel="stylesheet" href="/themes/custom/croydon/dist/assets/css/font-face.css">
-    
+    <link type="text/css" rel="stylesheet" href="/{{ base_path ~ directory }}/dist/assets/css/font-face.css">
+
     <css-placeholder token="{{ placeholder_token }}">
     <js-placeholder token="{{ placeholder_token }}">
-    
 
     <!--[if lt IE 9 ]>
-    <script src="/themes/custom/croydon/dist/assets/js/respond.min.js"></script>
+    <script src="{{ base_path ~ directory }}/dist/assets/js/respond.min.js"></script>
     <![endif]-->
 
     <!--[if lt IE 10]>
     <style>
     @media only screen and (min-width: 768px){
-    .services-list--multi li {
-    width: 33.33%;
-    float: left;
-    }
+      .services-list--multi li {
```

Comment:
I'm assuming this needs to be inline CSS, and it's needed for IE compatibility of output from the Services Modules? If so can inline CSS be injected by the Services modules? `[#attached]` or so?

Reply:
I think we can strip this. I wasn't actually concentrating on what it did. If it's needed later on we can inject it as you suggest.